### DISCUSSION TITLE
Google Authentication unauthorized

### DISCUSSION THREAD

**Discussion by pgi-jsanchez** [requarks/wiki#6148] [Error / Bug Report]
Steps to reproduce:
1. Follow the guide at (WikiJs documentation for Google SSO)[https://docs.requarks.io/auth/google]
2. Allow self registration for Google strategy
3. Open an incognito window and try to login using Google

Alternative staps:
a.1 Disable self registration for Google strategy
a.2 Create an user manually with Google stategy with the email
a.3 Try to login with that account using the Google SSO

b.1 Open security settings and disable "Block Open Redirect"
b.2 Try to login with a registered Google account

c.1  Open security settings and disable "Same Origin Referrer Policy" 
c.2 Try to login with a registered Google account

Current behavior:
WikiJs redirects the callback but not allows the login and display the unauthorized error page. Plus, no additional user is registered when I tested with self registration.

Expected behavior:
Login and registered the user if it's allowed.

At time of writing this message, the domain is allowed at Google Cloud Console, the redirect URI is assigned, scopes are there, Google plus API is enabled (I don't know if the change for Google Identity Services has something to do with this), tested with gsuite and non gsuite account.

![Captura de pantalla-20230209181025-499x219](https://user-images.githubusercontent.com/38993394/217967615-b2315a37-55ed-4186-bc96-8e093015f233.png)

![Captura de pantalla-20230209182953-425x298](https://user-images.githubusercontent.com/38993394/217969919-8ef1187a-c883-433d-81c0-470695491204.png)

Environment
Nginx reverse proxy with Letsencrypt (used for several deploys and works)
WikiJs version: 2.5.296
Postgres version: 11
Google Chrome: 109.0.5414.74
Firefox: 108.0.2

**Comment by mr-aybee** [comment]
When tried to add google auth as per mentioned in document,
![image](https://user-images.githubusercontent.com/39078635/234061528-f1652217-504c-41fc-ae5a-a3b25acd470a.png)
It says *Page not found* on trying google auth
